app/src/main/java/com/kickstarter/viewmodels/CreatorDashboardHeaderHolderViewModel.java

```diff
@@ -56,9 +56,7 @@ public ViewModel(final @NonNull Environment environment) {
 
       this.percentageFunded = this.projectAndStats
         .map(PairUtils::first)
-        .map(Project::percentageFunded)
-        .map(NumberUtils::flooredPercentage)
-        .compose(bindToLifecycle());
+        .map(p -> NumberUtils.flooredPercentage(p.percentageFunded()));
```

Comment:
nope, we don't need to `bindToLifecycle` when it's not a subscription to a subject i.e. when the output is an observable !